src/streamer.rs

```diff
@@ -623,7 +625,14 @@ pub fn broadcaster(
                     &sock,
                     &mut transmit_index,
                     &mut receive_index,
-                );
+                ) {
+                    match e {
+                        Error::RecvTimeoutError(RecvTimeoutError::Disconnected) => break,
+                        Error::RecvTimeoutError(RecvTimeoutError::Timeout) => (),
+                        Error::CrdtError(CrdtError::TooSmall) => (), // TODO: Why are the unit-tests throwing hundreds of these?
```

Comment:
the state transitions there are complicated.  you would need to boot, wait for the network to grow to N, then if it drops below MIN throw the error.